tardis/io/schemas/csvy_model.yml

```diff
@@ -0,0 +1,144 @@
+$schema: http://json-schema.org/draft-04/schema#
+type: object
+additionalProperties: false
+properties:
+  name: 
+    type: string
+    description: name of the model being run
+
+  description:
+    type: string
+    description: description of the model being run
+
+  tardis_model_config_version:
```

Comment:
that should be required

Reply:
tardis_model_config_version is now the only property required.